### PR TITLE
🎨 Video new styling #2354

### PR DESCRIPTION
**Pull request by BorghildSelle**

### CODE REVIEW

File: web/components/src/VideoJsPlayer/index.tsx [hunk 1]
```diff
@@ -63,6 +57,7 @@ export const VideoJS: React.FC<VideoJSProps> = ({
   }, [player])
 
   const getPlayer = (node: Element) => {
+    console.log('controls', controls)
```
Comment (millianapia) [inline web/components/src/VideoJsPlayer/index.tsx:60]: remove console.log

File: web/components/src/VideoJsPlayer/index.tsx [hunk 2]
```diff
@@ -63,6 +57,7 @@ export const VideoJS: React.FC<VideoJSProps> = ({
...
     const player = videojs(
```
Comment (padms) [inline web/components/src/VideoJsPlayer/index.tsx:61]: Clean up log

File: web/components/src/VideoJsPlayer/index.tsx (hunk 1 above)
Comment (padms) [inline web/components/src/VideoJsPlayer/index.tsx]: Clean up

**Review by millianapia — APPROVED** [review]
just remove console.log then it is good

**Review by padms — APPROVED** [review]

**Pull request merged**